src/tools/wasm2wat.cc

```diff
@@ -41,6 +41,7 @@ static Features s_features;
 static WriteWatOptions s_write_wat_options;
 static bool s_generate_names;
 static bool s_read_debug_names = true;
+static bool s_read_linking_names = false;
```

Comment:
Can you add this option on `wasm-decompile` too?

Reply:
Gladly!
Any other tool that would be interesting to have it as well?

Reply:
Turns out I actually missed adding the 6th option to ReadBinaryOptions in wasm-decompiler.cc, so I would have to modify it anyway. From my search here, I couldn't find any other file I would need to change.